pkg/controllers/networkinfo/vpcnetworkconfig_handler.go

```diff
@@ -54,6 +53,12 @@ func (h *VPCNetworkConfigurationHandler) Update(ctx context.Context, e event.Upd
 	log.V(1).Info("start processing VPC network config update event")
 	newNc := e.ObjectNew.(*v1alpha1.VPCNetworkConfiguration)
 
+	oldNc := e.ObjectOld.(*v1alpha1.VPCNetworkConfiguration)
+	if reflect.DeepEqual(oldNc.Spec, newNc.Spec) {
+		log.Info("Skip processing VPC network config update event", "newNc", newNc, "oldNc", oldNc)
+		return
+	}
+
```

Comment:
@heypnus for day2 logic, do we rely on NetworkConfiguration event to check conditions?

Reply:
I think what the day2 check depends on is the networkinfo event, not vpcnetworkconfiguration event.